README.md

```diff
@@ -23,6 +23,12 @@ If you're coming back to the application after a certain amount of time, you can
 script/bootstrap
 ```
 
+Copy the example environment variables across into a new .env file:
+
+```bash
+cp .env.example .env
+```
+
```

Comment:
Could we add this to the `script/setup` process? Something like:

```bash
if [ ! -f .env ]; then
  echo "==> Creating env file"
  cp .env.example .env
fi
```

Reply:
Ah it is already in the setup script, though not bootstrap as I expected 🤦‍♂️ I'll close.